components/ModSubmissionsEditFormFacility.vue

```diff
@@ -0,0 +1,252 @@
+            />
+            <ModInputField
+                data-testid="submission-form-addressLine2En"  
+                label="Address Line 2 En"
+                type="text"
+                placeholder="Apartment, building, room..."
+                :required="true"
+                :inputValidationCheck="validateAddressLineEn"
+                invalidInputErrorMessage="Invalid English Address"
+            />
+            <div class="flex flex-col mt-4">
+                <label for="Prefecture Japan" class="mb-2 text-zinc-900 text-sm font-bold font-['Noto Sans JP']">Prefecture Jp</label>
+                <select 
+                data-testid="submission-form-prefectureJp"
+                name="Prefecture Japan" id="1" class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400 text-neutral-600 text-sm font-normal font-['Noto Sans JP'] placeholder-gray-300"
+                v-model="store.prefectureJp"
+                >
+                    <option 
+                    v-for="(prefecture, index) in store.listPrefectureJapanJp" 
+                    :key="index"
+                    >
+                    {{ prefecture }}
+                    </option>
+                </select>
+            </div>
+            <ModInputField  
+                data-testid="submission-form-cityJp"
+                label="City Jp"
+                type="text"
+                placeholder="e.g. 渋谷区"
+                :required="true"
+                :inputValidationCheck="validateCityJp"
+                invalidInputErrorMessage="Invalid Japanese City Name"
+            />
+            <ModInputField  
+                data-testid="submission-form-addressLine1Jp"
+                label="Address Line 1 Jp"
+                type="text"
+                placeholder="Street address"
+                :required="true"
+                :inputValidationCheck="validateAddressLineJp"
+                invalidInputErrorMessage="Invalid Japanese Address"
+            />
+            <ModInputField  
+                data-testid="submission-form-addressLine2Jp"
+                label="Address Line 2 Jp"
+                type="text"
+                placeholder="Apartment, building, room..."
+                :required="true"
+                :inputValidationCheck="validateAddressLineJp"
+                invalidInputErrorMessage="Invalid Japanese Address"
+            />
+        </div>
+        
+        <div>
+            <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+                {{ $t('moderateSubmissionForm.googleMapsInformation') }}
+            </span>
+            <ModInputField
+                data-testid="submission-form-google-maps"  
+                label="Google maps URL"
+                type="url"
+                placeholder="e.g. https://www.google.co.jp/maps/"
+                :required="true"
+                :inputValidationCheck="validateWebsite"
```

Comment:
This validation check is here as 'placeholder' since Google URLs are very long the existing regex doesn't work for this field. At the moment when we submit it just checks for a general url, but that's all.